packages/react-app/src/services/auth.service.ts

```diff
@@ -93,6 +80,27 @@ export const createCache = async ({
 	return newCache;
 };
 
+export const getPermissions = async (accessToken: string, customerId: string): Promise<Role[]> => {
+	/**
+   * Returns a list of permissions for the current user
```

Comment:
I moved this here to make it much more obvious that permissions cached should wrap getPermissions. At the top of the file it's easy for a developer to assume that getPermissions should be used, and they will run into discord api limits